test/ConstructiveSolidGeometry/CSG_test.jl

```diff
@@ -22,13 +25,123 @@ using SolidStateDetectors.ConstructiveSolidGeometry:
+                    #@test !_in_angular_interval_open(T(3π), partial_2π)
+                    @test !_in_angular_interval_open(T(-π), partial_2π)
+                    @test _in_angular_interval_open(T(π/2), partial_2π)
+                    @test _in_angular_interval_open(T(-3π/2), partial_2π)
+                end
+                @testset "Negative Interval" begin
+                    neg_interval = -T(π/2)..T(π/2)
+                    @test _in_angular_interval_closed(T(0), neg_interval)
+                    @test _in_angular_interval_closed(T(2π), neg_interval)
+                    @test !_in_angular_interval_closed(T(π), neg_interval)
+                    @test !_in_angular_interval_closed(T(3π), neg_interval)
+                    @test !_in_angular_interval_closed(T(-π), neg_interval)
+                    @test _in_angular_interval_closed(T(-π/2), neg_interval)
+                    #@test _in_angular_interval_closed(T(-3π/2), neg_interval)
+                    @test _in_angular_interval_closed(T(3π/2), neg_interval)
+                    @test _in_angular_interval_closed(T(π/2), neg_interval)
+                    @test _in_angular_interval_open(T(0), neg_interval)
+                    @test !_in_angular_interval_open(T(π/2), neg_interval)
+                    @test !_in_angular_interval_open(T(-π/2), neg_interval)
+                    @test !_in_angular_interval_open(T(3π/2), neg_interval)
+                    @test !_in_angular_interval_open(T(-3π/2), neg_interval)
+                end
+            end
+
+            @testset "Surface Primitives" begin
+                @testset "ConalPlane" begin
+                    cplane = ConalPlane()
+                    @test CartesianPoint{T}(0, 0, 0) in cplane
+                    @test CartesianPoint{T}(1, 0, 0) in cplane
+                    @test CartesianPoint{T}(1, 0, 0.5) in cplane
+                    @test !(CartesianPoint{T}(1, 0.1, 0.5) in cplane)
+                end
+                @testset "ConeMantle" begin
+                    cmantle = ConeMantle(T(0.0), T(2.0), T(2.0))
+                    @test CartesianPoint{T}(0, 0, -1) in cmantle
+                    @test CartesianPoint{T}(2, 0, 1) in cmantle
+                    @test CartesianPoint{T}(0, 1, 0) in cmantle
+                    @test !(CartesianPoint{T}(0, 0, 0) in cmantle)
+                end
+                @testset "CylindricalAnnulus" begin
+                    cannulus = CylindricalAnnulus(T(2.0), T(0.0))
+                    @test CartesianPoint{T}(0, 0, 0) in cannulus
+                    @test CartesianPoint{T}(2, 0, 0) in cannulus
+                    @test CartesianPoint{T}(0, -2, 0) in cannulus
+                    @test !(CartesianPoint{T}(2, 2, 0) in cannulus)
+                    @test !(CartesianPoint{T}(0, 0, 0.1) in cannulus)
+                end
+                @testset "ToroidalAnnulus" begin
+                    tannulus = ToroidalAnnulus()
```

Comment:
Do we want `ToroidalAnnulus()` to have default values when nothing is passed?

Reply:
Ive made it so that all surface primitives have default values just like volume primitives. These are basically just convenience functions for fast testing

Reply:
Can you please still add the numbers (either in the function call or as a comment thereafter), so that one can easily see what the dimensions of the tested objects are without having to look for the default values in the source code?